backends/platform/sdl/win32/win32.cpp

```diff
@@ -150,6 +150,31 @@ bool OSystem_Win32::openUrl(const Common::String &url) {
 	return true;
 }
 
+Common::String OSystem_Win32::getSystemLanguage() const {
+#if defined(USE_DETECTLANG) && defined(USE_TRANSLATION)
+	// We can not use "setlocale" (at least not for MSVC builds), since it
+	// will return locales like: "English_USA.1252", thus we need a special
+	// way to determine the locale string for Win32.
+	char langName[9];
+	char ctryName[9];
+
+	const LCID languageIdentifier = GetUserDefaultUILanguage();
+
+	if (GetLocaleInfo(languageIdentifier, LOCALE_SISO639LANGNAME, langName, sizeof(langName)) != 0 &&
+		GetLocaleInfo(languageIdentifier, LOCALE_SISO3166CTRYNAME, ctryName, sizeof(ctryName)) != 0) {
+		Common::String localeName = langName;
+		localeName += "_";
+		localeName += ctryName;
+
+		return localeName;
+	} else {
+		return ModularBackend::getSystemLanguage();
+	}
+#endif // USE_DETECTLANG
```

Comment:
Maybe use `#else` here `#endif` after the return below?
If the `#if` above is true, all code paths return before reaching this point, so some compilers might complain about unreachable code with the code below.